packages/server/server.js

```diff
@@ -5,10 +5,12 @@ const session = require("express-session");
 const MongoStore = require("connect-mongo");
 const apiRoutes = require("./routes/apiRoutes");
 const path = require("path");
+const cors = require("cors");
```

Comment:
We shouldn't need cors when this is launched are you having troubles on development?